tests/common.py

```diff
@@ -1,6 +1,7 @@
 import unittest
 import numpy as np
 import jax
+jax.config.update("jax_enable_x64", True)
```

Comment:
Did we want to enable this here? Most of the tests have their own enable - and I'd rather for things that get imported to not silently include this

Reply:
Reverted in https://github.com/proteneer/timemachine/pull/453/commits/59fff43ddf47efafadcd3d3ec78b917a3e8c9b74 .

I missed if there's a way to set the Jax precision in a more fine-grained way.

In the current version of `test_nonbonded.py`, the precision is set to float64 [globally](https://github.com/proteneer/timemachine/blob/74efe28bfe4fe72a995a764bf3afe635b01bfc73/tests/test_nonbonded.py#L9). `compare_forces` sets the precision for the custom_ops implementation but [doesn't appear](https://github.com/proteneer/timemachine/blob/74efe28bfe4fe72a995a764bf3afe635b01bfc73/tests/common.py#L424-L448) to set the precision for the reference Jax implementation.